### PR TITLE
Update to newer aesophia

### PR DESCRIPTION
**Pull request by seanhinde**
This PR was sponsored by the ACF

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -16,8 +16,8 @@
 {profiles, [
     {local, []},
     {test, [
-        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"42cd47d"}}},
-                {aesophia_cli, {git, "git://github.com/aeternity/aesophia_cli", {tag, "v4.3.1"}}}]},
+        {deps, [{aesophia, {git, "https://github.com/aeternity/aesophia.git", {ref,"98a4049"}}},
```
Comment (uwiger) [inline rebar.config]: fe5f554 is the latest commit, as of 27 minutes ago

File: rebar.config [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
+                {aesophia_cli, {git, "git://github.com/aeternity/aesophia_cli", {ref, "45862a3"}}}]},
```
Comment (uwiger) [inline rebar.config]: This commit is from May 26. Why not a newer one?

**Review by uwiger — APPROVED** [review]

File: rebar.config (hunk 2 above)
Comment (seanhinde) [reply rebar.config]: Good question. Will take another look

File: rebar.config (hunk 1 above)
Comment (seanhinde) [reply rebar.config]: Aim. Fire. Delay. Miss :)

File: rebar.config (hunk 2 above)
Comment (seanhinde) [reply rebar.config]: Now at absolute latest

**Pull request merged**